docs/devGuide/maintainerGuide.md

```diff
@@ -77,30 +83,35 @@ You should see changes in `vue.min.js`, `vue-strap.js.map` and `vue-strap.min.js
 
 * **Note:** Do take a glance at the diff for `vue.min.js` to see if there's any strange changes (e.g. the version of jQuery changes even though no one upgraded it).
 
-2. Commit the new changes.
+3. Commit the new changes.
```

Comment:
Can actually use `1.` for all numbered lists in markdown: according to the [CommonMark](https://spec.commonmark.org/0.29/#ordered-list) specification, the subsequent ordered list numbers are disregarded.

Reply:
This doesn't seem to work for this file. I suspect it's because we have a bunch of other stuff between bullet points.

Reply:
Yes, the numbering will reset if the list is interrupted.